projects/daily_dribble_newsletter/index.html

```diff
@@ -2,25 +2,19 @@
 
 <html>
   <head>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
   </head>
   <body>
-    <h1>THE DAILY DRIBBLE</h1>
+    <div class="body">
 
-    <!-- Image credit: Thomas Serer https://unsplash.com/@jesusance -->
+    <h1 id="hr1">THE DAILY DRIBBLE</h1>
     <img src="images/stadium.jpg" />
+    <p id="p1">Your daily dose of soccer stories from on and off the pitch. <br>
+      Interviews, analysis, breaking news and more!</p> 
```

Comment:
The width of this paragraph element should not exceed that of the image above it.